deps.edn

```diff
@@ -25,7 +25,9 @@
                                com.bhauman/spell-spec {:mvn/version "0.1.2"}
                                org.clojure/spec-alpha2 {:git/url "https://github.com/clojure/spec-alpha2.git"
                                                         :sha "3d32b5e571b98e2930a7b2ed1dd9551bb269375a"}}}
-           :cljs-test-runner {:extra-deps {olical/cljs-test-runner {:mvn/version "3.8.0"}}
+           :cljs-test-runner {:extra-deps {olical/cljs-test-runner {:mvn/version "3.8.0"}
+                                           ; used only to pull in its externs file needed to compile js-joda types under advanced compilation
+                                           com.widdindustries/cljs.java-time {:mvn/version "0.1.20"}}
```

Comment:
Did you try using this namespace from a shadow-cljs project? It has better externs inference and will warn about cases where interop should use inference hints.

I might add shadow-cljs to Malli test setup just for inference checks.

Reply:
I think this was the only annotation needed for shadow-cljs
https://github.com/metosin/malli/pull/853/files#diff-d3e7f4bdb74b0d6cc969a8ef92f0f5dd63c43343c07e3d1347c2f636b43da117R83